src/main/java/com/aws/greengrass/util/Utils.java

```diff
@@ -610,7 +610,7 @@ public static void createPaths(Path... paths) throws IOException {
                 // This only supports POSIX compliant file permission right now. We will need to
                 // change this when trying to support Greengrass in Non-POSIX OS.
                 Files.createDirectories(p,
-                        PosixFilePermissions.asFileAttribute(PosixFilePermissions.fromString("rwx------")));
+                        PosixFilePermissions.asFileAttribute(PosixFilePermissions.fromString("rwxr-xr-x")));
```

Comment:
Is this the default permission and can it be overridden? I'm thinking about the cases when a directory needs to be read protected from all.

Reply:
We always set the permissions appropriately in `NucleusPaths`, but that class doesn't set the permissions on the parent directories like this one does.